docs/en/observability/monitor-infra/analyze-hosts.asciidoc

```diff
@@ -0,0 +1,170 @@
+backed by an easy-to-use interface called Lens. On the **Hosts** page, you can 
+view health and performance metrics to help you quickly:
+
+* Analyze and compare hosts without having to build new dashboards.
+//* Identify which hosts trigger the most alerts.
+* Troubleshoot and resolve issues quickly.
+//* View historical data to rule out false alerts and identify root causes.
+* Filter and search the data to focus on the hosts you care about the most.
+
+To access the **Hosts** page, go to **{observability} -> Infrastructure**, and
+then click **Hosts**. If the view is not enabled, click **Enable host view** to
+begin exploring.
+
+[role="screenshot"]
+image::images/hosts.png[Hosts page]
+
+The **Hosts** page provides a few different ways to view host metrics:
+
+* Overview tiles show the number of hosts returned by your search plus
+averages of key metrics, including CPU usage, memory usage, and throughput.
+* The host list shows all the hosts returned by your search with a
+breakdown of metrics for each host. You may need to page through the list or
+change the number of rows displayed on each page to see all of your hosts.
+* Each host name is an active link to an overview page with additional metrics
+about the host, such as CPU usage, load, memory usage, and network traffic.
+* The **Metrics** view shows metrics trending over time for the top 20 hosts,
+including normalized load, CPU usage, memory usage, network inbound, network
+outbound, disk read IOPS, and disk write IOPS. Place your cursor over a line to
+view metrics at a specific point in time.
+
+[NOTE]
+====
+This feature is in technical preview. We'd love to get your feedback!
+https://docs.google.com/forms/d/e/1FAIpQLScRHG8TIVb1Oq8ZhD4aks3P1TmgiM58TY123QpDCcBz83YC6w/viewform[Tell us what you think!]
+====
+
+[discrete]
+[[add-data-hosts-page]]
+== Add data
+
+The **Hosts** page displays metrics from indices that match the index patterns
+specified on the **Settings** page. For example, the default patterns are 
+`metrics-*` and `metricbeat-*`. 
+
+To add metrics:
+
+. Go to the **Hosts** page.
+. If there are no metrics to display, {kib} will prompt you to add an
+integration. Otherwise click **Add data**.
+. Search for an integration that matches the type of host you want to monitor
+and add the integration.
+. Follow the in-product instructions to install the integration plus any
+required programs, like {agent} or {metricbeat}.
+
+To learn more about adding observability data, refer to
+<<add-observability-data>>.
+
+[discrete]
+[[analyze-hosts-filter-view]]
+== Filter the Hosts view
+
+The **Hosts** page provides several mechanisms for filtering the data on the
+page:
+
+* Enter a search query to show metrics that match your search criteria. For
+example, to see metrics for all hosts running on Google Cloud Platform (GCP),
+enter `cloud.account.provider : "gcp"`. Otherwise you'll see metrics for all
```

Comment:
@roshan-elastic I need to replace this with a useful example and maybe add a couple more examples to this section. Any ideas?

Reply:
👍 Changed in 40b6ba4c1b4c613363b7e8b4b9106093c9c23cd3.